http/headers/Sec-CH-Viewport-Width.json

```
@@ -0,0 +1,38 @@
+{
+  "http": {
+    "headers": {
+      "Sec-CH-Viewport-Width": {
+        "__compat": {
+          "description": "`Sec-CH-Viewport-Width` request header",
+          "spec_url": "https://wicg.github.io/responsive-image-client-hints/#sec-ch-viewport-width",
+          "support": {
+            "chrome": {
+              "version_added": "97"
+            },
```

It looks Chrome already had `Viewport-Width` in Chrome 96 (didn't identify first version).

```suggestion
            "chrome": [
              {
                "version_added": "97"
              },
              {
                "version_added": "...",
                "alternative_name": "Viewport-Width"
              }
            ],
```